install.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+set -e
+# INSTALL DDB ON WSL WITH AN OFFICIAL UBUNTU VERSION
+
+# Check if the script is run as root if not rerun as root
+if [ "$(id -u)" != "0" ]; then
+    sudo $0
+    exit $?
+fi
+DDB_USER=${DDB_USER:-$SUDO_USER}
+
+if [ -z "$DDB_USER" ]; then
+    echo "DDB_USER is not defined or could not be detected from SUDO_USER. Set the DDB_USER variable, or run this script as your target user (must be in sudoer)."
+    exit 1
+fi
+
+#Update and upgrade packages
+apt-get update -y
+apt-get upgrade -y
+
+#Install and configuration of Docker
+## Add Docker's official GPG key:
+apt-get install ca-certificates curl make -y
+install -m 0755 -d /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
+chmod a+r /etc/apt/keyrings/docker.asc
+
+## Add the repository to Apt sources:
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu \
+  $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
+  tee /etc/apt/sources.list.d/docker.list > /dev/null
+
+apt-get update -y
+
+apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin -y
+
+## By default, the Docker daemon requires root privileges to run. To use Docker without sudo, add your user to the Docker group
+usermod -aG docker $DDB_USER
+
+## Expose docker to Windows by 2375 port create as root
+mkdir -p /etc/systemd/system/docker.service.d
+
+## Create override.conf file to expose docker to Windows
+### Define the string to check and add
+EXPOSE_DOCKER="[Service]\nExecStart=\nExecStart=/usr/bin/dockerd -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock"
+### Define the path to the override.conf file
+OVERRIDE_PATH="/etc/systemd/system/docker.service.d/override.conf"
+
+### Check if the file exist or the string exists in the override.conf file
+if [ ! -f "$OVERRIDE_PATH" ] || ! grep -Fxq "$EXPOSE_DOCKER" "$OVERRIDE_PATH"
+then
+    ### If the string does not exist, append it to the file
+    echo -e "$EXPOSE_DOCKER" >> "$OVERRIDE_PATH"
+fi
```

Comment:
not sure we actually need that : 

tcp exposure was made as a way for PHPStorm to run docker commands (by using the docker(-compose).exe and tcp URL of the daemon. Isn't there a WSL option nowadays where it can find the docker daemon instead ?